jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/datepicker/TypedDatePicker.java

```diff
@@ -98,6 +99,7 @@ public V getTypedValue() {
         return internalValue;
     }
 
+    @StudioIgnore
```

Comment:
I suppose that this annotation should be added to the methods within `SupportTypedValue` interface, because this  interface is implemented by multiple components, not only `DatePicker`.